contracts/ERC404.sol

```diff
@@ -125,6 +125,10 @@ abstract contract ERC404 is IERC404 {
     return _storedERC721Ids.length();
   }
 
+  function isValidValueOrId(uint256 valueOrId) internal view returns (bool) {
```

Comment:
since this function returns true if the passed value is a token id, i think it would be more appropriate to name it like this, and to use our underscore convention for internal functions:

```suggestion
  function _isTokenId(uint256 valueOrId) internal view returns (bool) {
```